apache-php/README.md

```diff
@@ -1,15 +1,12 @@
 # Apache and PHP on Docker
 
-This is a basic Docker environment for PHP development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php).
+[![Build Status](https://travis-ci.org/akeneo/Dockerfiles.svg?branch=php-7.1)](https://travis-ci.org/akeneo/Dockerfiles/tree/php-7.1)
 
-It is not intended to use directly as it stands, but rather to extend it and create custom development environments.
+This is a basic Docker environment for PHP development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php). **This image does not contain Akeneo PIM**.
 
-It provides a pre-configured Apache (2.4) + mod_php (PHP 7.1) web server, based on Debian 8 (Jessie).
-Apache is configured to run with the `docker` user. It is then preferred to place your PHP application in `/home/docker`
-and modify the default VHOST to access it.
+It provides a pre-configured Apache (2.4) + mod_php ( PHP 7.1 from [Sury repository](https://deb.sury.org/)) web server, based on Debian 9 (Stretch).
```

Comment:
sometimes you write pre-configured while you also write it "preconfigured". looks like both are okay, though you have to be consistent.